components/dealInDetails/DealDescription.tsx

```diff
@@ -0,0 +1,131 @@
+import { Box, Typography } from "@mui/material";
+import React, { useEffect } from "react";
+import theme from "../../theme";
+import { all_center } from "../../constant/commonStyle";
+import { useSelector } from "react-redux";
+import { useNavigate, useLocation } from "react-router-dom";
+
+const DealDescription = () => {
+  const dealAndStoreAllDetails = useSelector(
+    (store) => store.dealData.dealAndStoreAllDetails
+  );
+
+
+
+  const description = dealAndStoreAllDetails?.description;
+  const navigate = useNavigate();
+  const location = useLocation();
+
+  const myParam = new URLSearchParams(location.search).get('myParam');
+
+  
+
+  useEffect(() => {
+    if (description) {
+      document.querySelector("#innerHtml").innerHTML = description;
+    }
+  }, [description]);
+
+  return (
+    <Box
+      sx={{
+        ...all_center,
+        height: "auto",
+        width: "98%",
+        mt: { xl: "3rem" },
+        border: `1px solid ${theme.palette.grey[300]}`,
+        borderRadius: "10px",
+        flexDirection: "column",
+      }}
+    >
+      <Box
+        sx={{
+          height: "90%",
+          width: "90%",
+          display: "flex",
+          flexDirection: "column",
+          mt:{xl:"2rem"}
+        }}
+      >
+        <Typography
+          sx={{ justifySelf: "start", fontSize: theme.typography.h5.lg }}
```

Comment:
Sir in MUI theme inside typography i defined all the required font sizes to related variants so if give font size directly to typography it does not take that variant size, because of this  i give font size like this.